README.md

```diff
@@ -1,2 +1,25 @@
-# Hey-Micro
-My Hello Microverse Project
+# Hello-Micrverse
+
+This is my Hello Microverse project.
+
+### Built With
+Html<br />
+CSS
+
+### AUTHOR
+
+Inyambo Situmbeko
+
+<a href="https://github.com/BlakeYall">Github</a> 
+
+<a href="https://twitter.com/Thee_indigo1">Twitter</a> 
+
+<a href="https://www.linkedin.com/in/inyambo-situmbeko-524bb7229/">Linkedin</a>
```

Comment:
- [ ] Please add your social media links in a proper format :blush: because it will look great and will show your professionalism :+1:
**You can use this [template](https://github.com/HAMMAS-SALEEM/Hello-Microverse/blob/master/README.md#authors) to get a good idea** :+1: